src/test/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryServiceTest.java

```diff
@@ -200,6 +211,52 @@ public void summaryDryRunForceAddSameSourceFile() throws Exception {
         assertOutputMatches(".*New Files Mapped: +-1.*");
         assertOutputMatches(".*Total Files Mapped: +1.*");
         assertOutputMatches(".*Total Files in Project: +3.*");
+        assertOutputContains("25,," + path1);
+    }
+
+    @Test
+    public void summaryDryRunSampleSizeWithPreviousMappings() throws Exception {
+        testHelper.indexExportData("gilmer");
+        Path path1 = testHelper.addSourceFile("25.txt");
+        Path path2 = testHelper.addSourceFile("26.txt");
+        writeCsv(mappingBody("25,," + path1 +","));
+        writeTempCsv(mappingBody("25,," + path1 +",","26,," + path2 +",","27,," + path1 +",","28,," + path2 +",",
+                "29,," + path1 +",", "30,," + path2 +",", "31,," + path1 +","));
+
+        summaryService.capturePreviousState();
+        summaryService.setDryRun(true);
+        summaryService.setSampleSize(3);
+        summaryService.summary(Verbosity.NORMAL);
+
+        assertOutputMatches(".*Previous Files Mapped: +1.*");
+        assertOutputMatches(".*New Files Mapped: +6.*");
+        assertOutputMatches(".*Total Files Mapped: +7.*");
+        assertOutputMatches(".*Total Files in Project: +161.*");
+        assertOutputContains("26,," + path2);
+        assertOutputContains("28,," + path2);
+        assertOutputContains("30,," + path2);
```

Comment:
Might be good to add an assertion that the previous mapping doesn't show up in the output with `assertOutputDoesNotContain`.